doc/src/conf.py

```diff
@@ -363,26 +363,20 @@
 
 # Additional stuff for the LaTeX preamble.
 # Tweaked to work with XeTeX.
+latex_engine = 'xelatex'
 latex_elements = {
-    'babel':     '',
-    'fontenc': r'''
-% Define version of \LaTeX that is usable in math mode
-\let\OldLaTeX\LaTeX
-\renewcommand{\LaTeX}{\text{\OldLaTeX}}
-
+    'babel': r'\usepackage[english]{babel}',
+    'fontpkg': r'''
 \usepackage{bm}
-\usepackage{amssymb}
-\usepackage{fontspec}
-\usepackage[english]{babel}
 \defaultfontfeatures{Mapping=tex-text}
```

Comment:
This ``\defaultfontfeatures{Mapping=tex-text}`` should arguably be removed (cf https://github.com/sphinx-doc/sphinx/pull/6888) except if there is some good reason to keep it.

Reply:
I have to research more at some point this https://github.com/sphinx-doc/sphinx/pull/6888, DejaVu has `tlig` feature, I have to check what fontspec does which transforms `"` into curly double quotes which is what https://github.com/sphinx-doc/sphinx/pull/6888 does not want, and make the `Mapping=tex-text` no good, but perhaps we can salvage other ligatures such as `ff`.

Reply:
Just to be clear, DejaVu Sans Mono only needs to be used for the monospace font (code examples). This is because it is the only font that renders Unicode pretty printed output correctly (for instance, compare this with DejaVu Sans Mono and other fonts)

```
⌠
⎮     1
⎮ ───────── dx
⎮   _______
⎮ ╲╱ x + 1
⌡
```

See https://github.com/sympy/sympy/issues/15700.

For body and heading text, the font can be anything. It would probably be best to use whatever Sphinx defaults to, or whatever makes the math look good.

Reply:
Sphinx default to FreeSerif, FreeSans (GNU FreeFont).  The maths does look a bit small with DejaVu.  

I might try a FreeSerif build.  But in my fix #26883 to Missing characters I use that sans serif is DejaVu Sans.  This was some time-costly work and I am not eager to redo it from scratch to see what works with other font.